components/src/Checkbox/Checkbox.js

```diff
@@ -5,15 +5,18 @@ import theme from "../theme";
 import Box from "../Box/Box";
 import Text from "../Type/Text";
 
-const getFill = disabled => (disabled ? theme.colors.lightGrey : theme.colors.darkBlue);
+const getUncheckedBackgroundColour = disabled => (disabled ? theme.colors.whiteGrey : theme.colors.white);
```

Comment:
Would you consider cleaning this up like `inputStyles` in `Input`?

I can prepare a commit for you as well if you're interested. Let me know! :)

Reply:
I have edited the file. Can you have another look?